tests/functional/delete_runs/delete_runs_test_files/project_info.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "simple",
```

Comment:
I used this because this project contains only a `main.cpp` which has `int main() {}` in it. This is faster to build and check which is done 5 times. We only need the existence and number of runs and we don't need any results. Of course I can use the default project if needed.